persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/DataAccessTokenMapper.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+
+<mapper namespace="org.cbioportal.persistence.mybatis.DataAccessTokenMapper">
+    <cache readOnly="false"/>
+
+    <sql id="select">
+        data_access_tokens.TOKEN AS "${prefix}token",
```

Comment:
why need a ${prefix}?

Reply:
@sheridancbio can you comment on this? I can't remember what this was for... was it for if we wanted to encode a unique prefix to the front of the tokens? I remember a similar discussion where we decided against that because it was redundant given the nature of JWTs

Reply:
I don't understand why this is needed either. We will try removing the ${prefix} property here and also in the two named functions below. I think it should still work correctly. Thanks

Reply:
Looking further, @ao508 points out that this approach is used consistently in the cBioPortal code base for all mybatis mapper classes. We don't fully understand why it is necessary but suspect it has something to do with the framework which maps the responses from mysql onto constructed java objects and a need to have the fields returned from the sql query match the case of data member names in the model classes. For an example of this see : https://github.com/cBioPortal/cbioportal/blob/ff2c0499e09a7f608b389a301e30d7fb21ff9b44/persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/CancerTypeMapper.xml#L11

Reply:
Because all of the other mappers in the mybatis persistence module use this pattern, I think we prefer to stay consistent (even though we don't fully understand it)